[2024-02-26 20:47:30,047: INFO: main: Welcome to our custom log]
[2024-02-26 22:12:12,558: INFO: common: yaml file: config/config.yaml loaded successfully]
[2024-02-26 22:13:03,938: INFO: common: yaml file: config/config.yaml loaded successfully]
[2024-02-26 22:13:12,413: INFO: common: yaml file: config/config.yaml loaded successfully]
[2024-02-26 22:13:12,414: INFO: common: yaml file: params.yaml loaded successfully]
[2024-02-26 22:13:12,415: INFO: common: created directory at: artifacts]
[2024-02-26 22:13:12,415: INFO: common: created directory at: artifacts/data-ingestion]
[2024-02-26 22:16:16,265: INFO: common: yaml file: config/config.yaml loaded successfully]
[2024-02-26 22:16:16,266: INFO: common: yaml file: params.yaml loaded successfully]
[2024-02-26 22:16:16,267: INFO: common: created directory at: artifacts]
[2024-02-26 22:16:16,267: INFO: common: created directory at: artifacts/data-ingestion]
[2024-02-26 22:16:21,786: INFO: common: yaml file: config/config.yaml loaded successfully]
[2024-02-26 22:16:21,787: INFO: common: yaml file: params.yaml loaded successfully]
[2024-02-26 22:16:21,787: INFO: common: created directory at: artifacts]
[2024-02-26 22:16:21,788: INFO: common: created directory at: artifacts/data-ingestion]
[2024-02-26 22:16:48,133: INFO: common: yaml file: config/config.yaml loaded successfully]
[2024-02-26 22:16:48,134: INFO: common: yaml file: params.yaml loaded successfully]
[2024-02-26 22:16:48,135: INFO: common: created directory at: artifacts]
[2024-02-26 22:16:48,135: INFO: common: created directory at: artifacts/data-ingestion]
[2024-02-26 22:16:48,136: INFO: 1975804308: Downloading data from https://drive.google.com/file/d/1f2YFj0UGUtmvLvo81d9nF6PSyd45WcpN/view?usp=drive_link into file artifacts/data_ingestion/data.zip]
[2024-02-26 22:21:28,342: INFO: 1975804308: Downloaded data from https://drive.google.com/file/d/1f2YFj0UGUtmvLvo81d9nF6PSyd45WcpN/view?usp=drive_link into file artifacts/data_ingestion/data.zip]
[2024-02-26 22:36:07,617: INFO: main: >>>>>> stage Data Ingestion stage started <<<<<<]
[2024-02-26 22:36:07,618: INFO: common: yaml file: config/config.yaml loaded successfully]
[2024-02-26 22:36:07,618: INFO: common: yaml file: params.yaml loaded successfully]
[2024-02-26 22:36:07,618: INFO: common: created directory at: artifacts]
[2024-02-26 22:36:07,619: INFO: common: created directory at: artifacts/data-ingestion]
[2024-02-26 22:36:07,619: INFO: data_ingestion: Downloading data from https://drive.google.com/file/d/1f2YFj0UGUtmvLvo81d9nF6PSyd45WcpN/view?usp=drive_link into file artifacts/data_ingestion/data.zip]
[2024-02-26 22:40:47,255: INFO: data_ingestion: Downloaded data from https://drive.google.com/file/d/1f2YFj0UGUtmvLvo81d9nF6PSyd45WcpN/view?usp=drive_link into file artifacts/data_ingestion/data.zip]
[2024-02-26 22:40:55,074: INFO: main: >>>>>> stage Data Ingestion stage completed <<<<<<

x=============x]
[2024-02-27 01:47:35,394: INFO: common: yaml file: config/config.yaml loaded successfully]
[2024-02-27 01:47:35,396: INFO: common: yaml file: params.yaml loaded successfully]
[2024-02-27 01:47:35,396: INFO: common: created directory at: artifacts]
[2024-02-27 01:47:35,397: INFO: common: created directory at: artifacts/prepare_base_model]
[2024-02-27 01:47:43,520: WARNING: saving_utils: Compiled the loaded model, but the compiled metrics have yet to be built. `model.compile_metrics` will be empty until you train or evaluate the model.]
[2024-02-27 01:48:19,988: INFO: common: yaml file: config/config.yaml loaded successfully]
[2024-02-27 01:48:19,989: INFO: common: yaml file: params.yaml loaded successfully]
[2024-02-27 01:48:19,990: INFO: common: created directory at: artifacts]
[2024-02-27 01:48:19,991: INFO: common: created directory at: artifacts/prepare_base_model]
[2024-02-27 01:48:20,197: WARNING: saving_utils: Compiled the loaded model, but the compiled metrics have yet to be built. `model.compile_metrics` will be empty until you train or evaluate the model.]
[2024-02-28 03:11:37,226: INFO: common: yaml file: config/config.yaml loaded successfully]
[2024-02-28 03:11:37,227: INFO: common: yaml file: params.yaml loaded successfully]
[2024-02-28 03:11:37,228: INFO: common: created directory at: artifacts]
[2024-02-28 03:11:37,229: INFO: common: created directory at: artifacts/training]
[2024-02-28 03:22:30,583: INFO: common: yaml file: config/config.yaml loaded successfully]
[2024-02-28 03:22:30,585: INFO: common: yaml file: params.yaml loaded successfully]
[2024-02-28 03:22:30,585: INFO: common: created directory at: artifacts]
[2024-02-28 03:22:30,586: INFO: common: created directory at: artifacts/training]
[2024-02-28 04:27:25,260: INFO: main: >>>>>> stage Data Ingestion stage started <<<<<<]
[2024-02-28 04:27:25,262: INFO: common: yaml file: config/config.yaml loaded successfully]
[2024-02-28 04:27:25,263: INFO: common: yaml file: params.yaml loaded successfully]
[2024-02-28 04:27:25,263: INFO: common: created directory at: artifacts]
[2024-02-28 04:27:25,263: INFO: common: created directory at: artifacts/data-ingestion]
[2024-02-28 04:27:25,263: INFO: data_ingestion: Downloading data from https://drive.google.com/file/d/1f2YFj0UGUtmvLvo81d9nF6PSyd45WcpN/view?usp=drive_link into file artifacts/data_ingestion/data.zip]
[2024-02-28 04:28:53,821: INFO: data_ingestion: Downloaded data from https://drive.google.com/file/d/1f2YFj0UGUtmvLvo81d9nF6PSyd45WcpN/view?usp=drive_link into file artifacts/data_ingestion/data.zip]
[2024-02-28 04:29:02,474: INFO: main: >>>>>> stage Data Ingestion stage completed <<<<<<

x=============x]
[2024-02-28 04:29:02,474: INFO: main: *******************]
[2024-02-28 04:29:02,474: INFO: main: >>>>>> stage Prepare base model started]
[2024-02-28 04:29:02,476: INFO: common: yaml file: config/config.yaml loaded successfully]
[2024-02-28 04:29:02,477: INFO: common: yaml file: params.yaml loaded successfully]
[2024-02-28 04:29:02,477: INFO: common: created directory at: artifacts]
[2024-02-28 04:29:02,477: INFO: common: created directory at: artifacts/prepare_base_model]
[2024-02-28 04:29:10,108: WARNING: saving_utils: Compiled the loaded model, but the compiled metrics have yet to be built. `model.compile_metrics` will be empty until you train or evaluate the model.]
[2024-02-28 04:29:10,276: INFO: main: >>>>>> stage Prepare base model completed <<<<<<

x==============x]
[2024-02-28 04:29:10,276: INFO: main: *******************]
[2024-02-28 04:29:10,276: INFO: main: >>>>>> stage Model Training started]
[2024-02-28 04:29:10,278: INFO: common: yaml file: config/config.yaml loaded successfully]
[2024-02-28 04:29:10,279: INFO: common: yaml file: params.yaml loaded successfully]
[2024-02-28 04:29:10,279: INFO: common: created directory at: artifacts]
[2024-02-28 04:29:10,279: ERROR: main: name 'os' is not defined]
Traceback (most recent call last):
  File "main.py", line 33, in <module>
    obj.main()
  File "/scratch/Shaz/subsidiary_work/Kidney_Disease_Classification/src/cnn_classifier/pipeline/stage_03_model_training.py", line 13, in main
    training_config = config.get_training_config()
  File "/scratch/Shaz/subsidiary_work/Kidney_Disease_Classification/src/cnn_classifier/config/configuration.py", line 52, in get_training_config
    training_data = os.path.join(self.config.data_ingestion.unzip_dir, "kidney-ct-scan-image")
NameError: name 'os' is not defined
[2024-02-28 04:31:53,468: INFO: main: >>>>>> stage Data Ingestion stage started <<<<<<]
[2024-02-28 04:31:53,470: INFO: common: yaml file: config/config.yaml loaded successfully]
[2024-02-28 04:31:53,471: INFO: common: yaml file: params.yaml loaded successfully]
[2024-02-28 04:31:53,471: INFO: common: created directory at: artifacts]
[2024-02-28 04:31:53,471: INFO: common: created directory at: artifacts/data-ingestion]
[2024-02-28 04:31:53,471: INFO: data_ingestion: Downloading data from https://drive.google.com/file/d/1f2YFj0UGUtmvLvo81d9nF6PSyd45WcpN/view?usp=drive_link into file artifacts/data_ingestion/data.zip]
[2024-02-28 04:32:32,112: INFO: data_ingestion: Downloaded data from https://drive.google.com/file/d/1f2YFj0UGUtmvLvo81d9nF6PSyd45WcpN/view?usp=drive_link into file artifacts/data_ingestion/data.zip]
[2024-02-28 04:32:55,297: INFO: main: >>>>>> stage Data Ingestion stage completed <<<<<<

x=============x]
[2024-02-28 04:32:55,297: INFO: main: *******************]
[2024-02-28 04:32:55,297: INFO: main: >>>>>> stage Prepare base model started]
[2024-02-28 04:32:55,299: INFO: common: yaml file: config/config.yaml loaded successfully]
[2024-02-28 04:32:55,300: INFO: common: yaml file: params.yaml loaded successfully]
[2024-02-28 04:32:55,300: INFO: common: created directory at: artifacts]
[2024-02-28 04:32:55,300: INFO: common: created directory at: artifacts/prepare_base_model]
[2024-02-28 04:32:57,110: WARNING: saving_utils: Compiled the loaded model, but the compiled metrics have yet to be built. `model.compile_metrics` will be empty until you train or evaluate the model.]
[2024-02-28 04:32:57,291: INFO: main: >>>>>> stage Prepare base model completed <<<<<<

x==============x]
[2024-02-28 04:32:57,291: INFO: main: *******************]
[2024-02-28 04:32:57,291: INFO: main: >>>>>> stage Model Training started]
[2024-02-28 04:32:57,292: INFO: common: yaml file: config/config.yaml loaded successfully]
[2024-02-28 04:32:57,293: INFO: common: yaml file: params.yaml loaded successfully]
[2024-02-28 04:32:57,293: INFO: common: created directory at: artifacts]
[2024-02-28 04:32:57,294: INFO: common: created directory at: artifacts/training]
[2024-02-28 04:32:57,569: ERROR: main: [Errno 2] No such file or directory: 'artifacts/data_ingestion/kidney-ct-scan-image']
Traceback (most recent call last):
  File "main.py", line 33, in <module>
    obj.main()
  File "/scratch/Shaz/subsidiary_work/Kidney_Disease_Classification/src/cnn_classifier/pipeline/stage_03_model_training.py", line 16, in main
    training.train_valid_generator()
  File "/scratch/Shaz/subsidiary_work/Kidney_Disease_Classification/src/cnn_classifier/components/model_training.py", line 36, in train_valid_generator
    self.valid_generator = valid_datagenerator.flow_from_directory(
  File "/scratch/Shaz/miniconda3/envs/kidney/lib/python3.8/site-packages/keras/preprocessing/image.py", line 1648, in flow_from_directory
    return DirectoryIterator(
  File "/scratch/Shaz/miniconda3/envs/kidney/lib/python3.8/site-packages/keras/preprocessing/image.py", line 563, in __init__
    for subdir in sorted(os.listdir(directory)):
FileNotFoundError: [Errno 2] No such file or directory: 'artifacts/data_ingestion/kidney-ct-scan-image'
[2024-02-28 04:35:18,683: INFO: main: >>>>>> stage Data Ingestion stage started <<<<<<]
[2024-02-28 04:35:18,685: INFO: common: yaml file: config/config.yaml loaded successfully]
[2024-02-28 04:35:18,686: INFO: common: yaml file: params.yaml loaded successfully]
[2024-02-28 04:35:18,686: INFO: common: created directory at: artifacts]
[2024-02-28 04:35:18,686: INFO: common: created directory at: artifacts/data-ingestion]
[2024-02-28 04:35:18,686: INFO: data_ingestion: Downloading data from https://drive.google.com/file/d/1f2YFj0UGUtmvLvo81d9nF6PSyd45WcpN/view?usp=drive_link into file artifacts/data_ingestion/data.zip]
[2024-02-28 04:35:55,566: INFO: data_ingestion: Downloaded data from https://drive.google.com/file/d/1f2YFj0UGUtmvLvo81d9nF6PSyd45WcpN/view?usp=drive_link into file artifacts/data_ingestion/data.zip]
[2024-02-28 04:36:04,362: INFO: main: >>>>>> stage Data Ingestion stage completed <<<<<<

x=============x]
[2024-02-28 04:36:04,362: INFO: main: *******************]
[2024-02-28 04:36:04,362: INFO: main: >>>>>> stage Prepare base model started]
[2024-02-28 04:36:04,364: INFO: common: yaml file: config/config.yaml loaded successfully]
[2024-02-28 04:36:04,365: INFO: common: yaml file: params.yaml loaded successfully]
[2024-02-28 04:36:04,365: INFO: common: created directory at: artifacts]
[2024-02-28 04:36:04,365: INFO: common: created directory at: artifacts/prepare_base_model]
[2024-02-28 04:36:06,343: WARNING: saving_utils: Compiled the loaded model, but the compiled metrics have yet to be built. `model.compile_metrics` will be empty until you train or evaluate the model.]
[2024-02-28 04:36:06,556: INFO: main: >>>>>> stage Prepare base model completed <<<<<<

x==============x]
[2024-02-28 04:36:06,556: INFO: main: *******************]
[2024-02-28 04:36:06,556: INFO: main: >>>>>> stage Model Training started]
[2024-02-28 04:36:06,557: INFO: common: yaml file: config/config.yaml loaded successfully]
[2024-02-28 04:36:06,558: INFO: common: yaml file: params.yaml loaded successfully]
[2024-02-28 04:36:06,559: INFO: common: created directory at: artifacts]
[2024-02-28 04:36:06,559: INFO: common: created directory at: artifacts/training]
[2024-02-28 04:41:22,693: INFO: main: >>>>>> stage Data Ingestion stage started <<<<<<]
[2024-02-28 04:41:22,695: INFO: common: yaml file: config/config.yaml loaded successfully]
[2024-02-28 04:41:22,696: INFO: common: yaml file: params.yaml loaded successfully]
[2024-02-28 04:41:22,696: INFO: common: created directory at: artifacts]
[2024-02-28 04:41:22,696: INFO: common: created directory at: artifacts/data-ingestion]
[2024-02-28 04:41:22,696: INFO: data_ingestion: Downloading data from https://drive.google.com/file/d/1f2YFj0UGUtmvLvo81d9nF6PSyd45WcpN/view?usp=drive_link into file artifacts/data_ingestion/data.zip]
[2024-02-28 04:41:58,738: INFO: data_ingestion: Downloaded data from https://drive.google.com/file/d/1f2YFj0UGUtmvLvo81d9nF6PSyd45WcpN/view?usp=drive_link into file artifacts/data_ingestion/data.zip]
[2024-02-28 04:42:22,950: INFO: main: >>>>>> stage Data Ingestion stage completed <<<<<<

x=============x]
[2024-02-28 04:42:22,950: INFO: main: *******************]
[2024-02-28 04:42:22,950: INFO: main: >>>>>> stage Prepare base model started]
[2024-02-28 04:42:22,952: INFO: common: yaml file: config/config.yaml loaded successfully]
[2024-02-28 04:42:22,952: INFO: common: yaml file: params.yaml loaded successfully]
[2024-02-28 04:42:22,953: INFO: common: created directory at: artifacts]
[2024-02-28 04:42:22,953: INFO: common: created directory at: artifacts/prepare_base_model]
[2024-02-28 04:42:24,875: WARNING: saving_utils: Compiled the loaded model, but the compiled metrics have yet to be built. `model.compile_metrics` will be empty until you train or evaluate the model.]
[2024-02-28 04:42:25,056: INFO: main: >>>>>> stage Prepare base model completed <<<<<<

x==============x]
[2024-02-28 04:42:25,056: INFO: main: *******************]
[2024-02-28 04:42:25,056: INFO: main: >>>>>> stage Model Training started]
[2024-02-28 04:42:25,057: INFO: common: yaml file: config/config.yaml loaded successfully]
[2024-02-28 04:42:25,058: INFO: common: yaml file: params.yaml loaded successfully]
[2024-02-28 04:42:25,059: INFO: common: created directory at: artifacts]
[2024-02-28 04:42:25,059: INFO: common: created directory at: artifacts/training]
[2024-02-28 16:15:12,868: INFO: main: >>>>>> stage Model Training completed <<<<<<

x==============x]
<<<<<<<< HEAD
[2024-02-29 04:03:31,849: INFO: main: >>>>>> stage Data Ingestion stage started <<<<<<]
[2024-02-29 04:03:31,851: INFO: common: yaml file: config/config.yaml loaded successfully]
[2024-02-29 04:03:31,852: INFO: common: yaml file: params.yaml loaded successfully]
[2024-02-29 04:03:31,852: INFO: common: created directory at: artifacts]
[2024-02-29 04:03:31,852: INFO: common: created directory at: artifacts/data-ingestion]
[2024-02-29 04:03:31,852: INFO: data_ingestion: Downloading data from https://drive.google.com/file/d/1f2YFj0UGUtmvLvo81d9nF6PSyd45WcpN/view?usp=drive_link into file artifacts/data_ingestion/data.zip]
[2024-02-29 04:05:01,985: INFO: data_ingestion: Downloaded data from https://drive.google.com/file/d/1f2YFj0UGUtmvLvo81d9nF6PSyd45WcpN/view?usp=drive_link into file artifacts/data_ingestion/data.zip]
[2024-02-29 04:05:16,159: INFO: main: >>>>>> stage Data Ingestion stage completed <<<<<<

x=============x]
[2024-02-29 04:05:16,159: INFO: main: *******************]
[2024-02-29 04:05:16,159: INFO: main: >>>>>> stage Prepare base model started]
[2024-02-29 04:05:16,161: INFO: common: yaml file: config/config.yaml loaded successfully]
[2024-02-29 04:05:16,162: INFO: common: yaml file: params.yaml loaded successfully]
[2024-02-29 04:05:16,162: INFO: common: created directory at: artifacts]
[2024-02-29 04:05:16,162: INFO: common: created directory at: artifacts/prepare_base_model]
[2024-02-29 04:05:18,124: WARNING: saving_utils: Compiled the loaded model, but the compiled metrics have yet to be built. `model.compile_metrics` will be empty until you train or evaluate the model.]
[2024-02-29 04:05:18,299: INFO: main: >>>>>> stage Prepare base model completed <<<<<<

x==============x]
[2024-02-29 04:05:18,299: INFO: main: *******************]
[2024-02-29 04:05:18,299: INFO: main: >>>>>> stage Model Training started]
[2024-02-29 04:05:18,301: INFO: common: yaml file: config/config.yaml loaded successfully]
[2024-02-29 04:05:18,302: INFO: common: yaml file: params.yaml loaded successfully]
[2024-02-29 04:05:18,302: INFO: common: created directory at: artifacts]
[2024-02-29 04:05:18,302: INFO: common: created directory at: artifacts/training]
[2024-02-29 15:04:50,504: INFO: main: >>>>>> stage Model Training completed <<<<<<

x==============x]

[2024-02-29 00:40:37,349: INFO: common: yaml file: config/config.yaml loaded successfully]
[2024-02-29 00:40:37,351: INFO: common: yaml file: params.yaml loaded successfully]
[2024-02-29 00:40:37,352: INFO: common: created directory at: artifacts]
[2024-02-29 00:41:10,433: INFO: common: yaml file: config/config.yaml loaded successfully]
[2024-02-29 00:41:10,435: INFO: common: yaml file: params.yaml loaded successfully]
[2024-02-29 00:41:10,436: INFO: common: created directory at: artifacts]
[2024-02-29 00:45:29,264: INFO: common: json file saved at: scores.json]
[2024-02-29 01:15:41,970: INFO: common: yaml file: config/config.yaml loaded successfully]
[2024-02-29 01:15:41,978: INFO: common: yaml file: params.yaml loaded successfully]
[2024-02-29 01:15:41,979: INFO: common: created directory at: artifacts]
<<<<<<< HEAD

=======
[2024-02-29 15:50:30,160: INFO: common: yaml file: config/config.yaml loaded successfully]
[2024-02-29 15:50:30,163: INFO: common: yaml file: params.yaml loaded successfully]
[2024-02-29 15:50:30,163: INFO: common: created directory at: artifacts]
[2024-02-29 15:54:50,451: INFO: common: json file saved at: scores.json]
[2024-02-29 15:56:41,023: INFO: common: yaml file: config/config.yaml loaded successfully]
[2024-02-29 15:56:41,025: INFO: common: yaml file: params.yaml loaded successfully]
[2024-02-29 15:56:41,025: INFO: common: created directory at: artifacts]
[2024-02-29 16:00:55,874: INFO: common: json file saved at: scores.json]
[2024-02-29 16:08:28,523: INFO: common: yaml file: config/config.yaml loaded successfully]
[2024-02-29 16:08:28,525: INFO: common: yaml file: params.yaml loaded successfully]
[2024-02-29 16:08:28,525: INFO: common: created directory at: artifacts]
[2024-02-29 16:12:37,294: INFO: common: json file saved at: scores.json]
[2024-02-29 16:37:12,554: INFO: common: yaml file: config/config.yaml loaded successfully]
[2024-02-29 16:37:12,555: INFO: common: yaml file: params.yaml loaded successfully]
[2024-02-29 16:37:12,556: INFO: common: created directory at: artifacts]
[2024-02-29 16:41:27,018: INFO: common: json file saved at: scores.json]
[2024-02-29 16:41:29,532: WARNING: save: Found untraced functions such as _jit_compiled_convolution_op, _jit_compiled_convolution_op, _jit_compiled_convolution_op, _jit_compiled_convolution_op, _jit_compiled_convolution_op while saving (showing 5 of 14). These functions will not be directly callable after loading.]
[2024-02-29 16:41:29,839: INFO: builder_impl: Assets written to: /tmp/tmpvu22rb6d/model/data/model/assets]
>>>>>>> 58f4dfce43ac14574c8d076926a0696f0c99d0e3
<<<<<<< HEAD
[2024-02-29 22:42:35,693: INFO: main: >>>>>> stage Data Ingestion stage started <<<<<<]
[2024-02-29 22:42:35,695: INFO: common: yaml file: config/config.yaml loaded successfully]
[2024-02-29 22:42:35,696: INFO: common: yaml file: params.yaml loaded successfully]
[2024-02-29 22:42:35,696: INFO: common: created directory at: artifacts]
[2024-02-29 22:42:35,696: INFO: common: created directory at: artifacts/data-ingestion]
[2024-02-29 22:42:35,696: INFO: data_ingestion: Downloading data from https://drive.google.com/file/d/1f2YFj0UGUtmvLvo81d9nF6PSyd45WcpN/view?usp=drive_link into file artifacts/data_ingestion/data.zip]
[2024-02-29 22:44:02,387: INFO: data_ingestion: Downloaded data from https://drive.google.com/file/d/1f2YFj0UGUtmvLvo81d9nF6PSyd45WcpN/view?usp=drive_link into file artifacts/data_ingestion/data.zip]
[2024-02-29 22:44:27,322: INFO: main: >>>>>> stage Data Ingestion stage completed <<<<<<

x=============x]
[2024-02-29 22:44:27,322: INFO: main: *******************]
[2024-02-29 22:44:27,322: INFO: main: >>>>>> stage Prepare base model started]
[2024-02-29 22:44:27,324: INFO: common: yaml file: config/config.yaml loaded successfully]
[2024-02-29 22:44:27,325: INFO: common: yaml file: params.yaml loaded successfully]
[2024-02-29 22:44:27,325: INFO: common: created directory at: artifacts]
[2024-02-29 22:44:27,325: INFO: common: created directory at: artifacts/prepare_base_model]
[2024-02-29 22:44:29,231: WARNING: saving_utils: Compiled the loaded model, but the compiled metrics have yet to be built. `model.compile_metrics` will be empty until you train or evaluate the model.]
[2024-02-29 22:44:29,493: INFO: main: >>>>>> stage Prepare base model completed <<<<<<

x==============x]
[2024-02-29 22:44:29,493: INFO: main: *******************]
[2024-02-29 22:44:29,493: INFO: main: >>>>>> stage Model Training started]
[2024-02-29 22:44:29,495: INFO: common: yaml file: config/config.yaml loaded successfully]
[2024-02-29 22:44:29,496: INFO: common: yaml file: params.yaml loaded successfully]
[2024-02-29 22:44:29,496: INFO: common: created directory at: artifacts]
[2024-02-29 22:44:29,496: INFO: common: created directory at: artifacts/training]
=======
[2024-03-01 13:50:16,225: INFO: _internal: [31m[1mWARNING: This is a development server. Do not use it in a production deployment. Use a production WSGI server instead.[0m
 * Running on all addresses (0.0.0.0)
 * Running on http://127.0.0.1:8080
 * Running on http://10.2.12.16:8080]
[2024-03-01 13:50:16,225: INFO: _internal: [33mPress CTRL+C to quit[0m]
[2024-03-01 13:50:51,667: INFO: _internal: 127.0.0.1 - - [01/Mar/2024 13:50:51] "GET / HTTP/1.1" 200 -]
[2024-03-01 13:51:34,020: INFO: _internal: 127.0.0.1 - - [01/Mar/2024 13:51:34] "GET / HTTP/1.1" 200 -]
[2024-03-01 13:54:02,193: INFO: _internal: [31m[1mWARNING: This is a development server. Do not use it in a production deployment. Use a production WSGI server instead.[0m
 * Running on all addresses (0.0.0.0)
 * Running on http://127.0.0.1:8080
 * Running on http://10.2.12.16:8080]
[2024-03-01 13:54:02,193: INFO: _internal: [33mPress CTRL+C to quit[0m]
[2024-03-01 13:54:09,809: INFO: _internal: 127.0.0.1 - - [01/Mar/2024 13:54:09] "GET / HTTP/1.1" 200 -]
[2024-03-01 13:54:12,683: INFO: _internal: 127.0.0.1 - - [01/Mar/2024 13:54:12] "[33mGET /favicon.ico HTTP/1.1[0m" 404 -]
[2024-03-01 13:56:26,361: INFO: _internal: [31m[1mWARNING: This is a development server. Do not use it in a production deployment. Use a production WSGI server instead.[0m
 * Running on all addresses (0.0.0.0)
 * Running on http://127.0.0.1:8080
 * Running on http://10.2.12.16:8080]
[2024-03-01 13:56:26,362: INFO: _internal: [33mPress CTRL+C to quit[0m]
[2024-03-01 13:56:28,132: INFO: _internal: 127.0.0.1 - - [01/Mar/2024 13:56:28] "GET / HTTP/1.1" 200 -]
[2024-03-01 13:56:28,150: INFO: _internal: 127.0.0.1 - - [01/Mar/2024 13:56:28] "[33mGET /kidney-image.jpg HTTP/1.1[0m" 404 -]
[2024-03-01 13:56:29,771: INFO: _internal: 127.0.0.1 - - [01/Mar/2024 13:56:29] "GET / HTTP/1.1" 200 -]
[2024-03-01 13:56:29,785: INFO: _internal: 127.0.0.1 - - [01/Mar/2024 13:56:29] "[33mGET /kidney-image.jpg HTTP/1.1[0m" 404 -]
[2024-03-01 13:56:45,207: INFO: _internal: 127.0.0.1 - - [01/Mar/2024 13:56:45] "GET / HTTP/1.1" 200 -]
[2024-03-01 13:56:45,220: INFO: _internal: 127.0.0.1 - - [01/Mar/2024 13:56:45] "[33mGET /kidney-image.jpg HTTP/1.1[0m" 404 -]
[2024-03-01 13:57:57,687: INFO: _internal: [31m[1mWARNING: This is a development server. Do not use it in a production deployment. Use a production WSGI server instead.[0m
 * Running on all addresses (0.0.0.0)
 * Running on http://127.0.0.1:8080
 * Running on http://10.2.12.16:8080]
[2024-03-01 13:57:57,687: INFO: _internal: [33mPress CTRL+C to quit[0m]
[2024-03-01 13:57:59,670: INFO: _internal: 127.0.0.1 - - [01/Mar/2024 13:57:59] "GET / HTTP/1.1" 200 -]
[2024-03-01 13:57:59,700: INFO: _internal: 127.0.0.1 - - [01/Mar/2024 13:57:59] "[33mGET /kidney-image.jpg HTTP/1.1[0m" 404 -]
[2024-03-01 13:57:59,880: INFO: _internal: 127.0.0.1 - - [01/Mar/2024 13:57:59] "[33mGET /kidney-image.jpg HTTP/1.1[0m" 404 -]
[2024-03-01 13:59:55,447: INFO: _internal: [31m[1mWARNING: This is a development server. Do not use it in a production deployment. Use a production WSGI server instead.[0m
 * Running on all addresses (0.0.0.0)
 * Running on http://127.0.0.1:8080
 * Running on http://10.2.12.16:8080]
[2024-03-01 13:59:55,447: INFO: _internal: [33mPress CTRL+C to quit[0m]
[2024-03-01 13:59:56,957: INFO: _internal: 127.0.0.1 - - [01/Mar/2024 13:59:56] "GET / HTTP/1.1" 200 -]
[2024-03-01 13:59:56,974: INFO: _internal: 127.0.0.1 - - [01/Mar/2024 13:59:56] "[33mGET /kidney-image.jpg HTTP/1.1[0m" 404 -]
[2024-03-01 13:59:57,273: INFO: _internal: 127.0.0.1 - - [01/Mar/2024 13:59:57] "[33mGET /kidney-image.jpg HTTP/1.1[0m" 404 -]
[2024-03-01 14:00:47,034: INFO: _internal: [31m[1mWARNING: This is a development server. Do not use it in a production deployment. Use a production WSGI server instead.[0m
 * Running on all addresses (0.0.0.0)
 * Running on http://127.0.0.1:8080
 * Running on http://10.2.12.16:8080]
[2024-03-01 14:00:47,034: INFO: _internal: [33mPress CTRL+C to quit[0m]
[2024-03-01 14:00:48,212: INFO: _internal: 127.0.0.1 - - [01/Mar/2024 14:00:48] "GET / HTTP/1.1" 200 -]
[2024-03-01 14:00:48,245: INFO: _internal: 127.0.0.1 - - [01/Mar/2024 14:00:48] "[33mGET /kidney-image.jpg HTTP/1.1[0m" 404 -]
[2024-03-01 14:00:48,558: INFO: _internal: 127.0.0.1 - - [01/Mar/2024 14:00:48] "[33mGET /kidney-image.jpg HTTP/1.1[0m" 404 -]
[2024-03-01 14:03:33,153: INFO: _internal: [31m[1mWARNING: This is a development server. Do not use it in a production deployment. Use a production WSGI server instead.[0m
 * Running on all addresses (0.0.0.0)
 * Running on http://127.0.0.1:8080
 * Running on http://10.2.12.16:8080]
[2024-03-01 14:03:33,153: INFO: _internal: [33mPress CTRL+C to quit[0m]
[2024-03-01 14:03:34,029: INFO: _internal: 127.0.0.1 - - [01/Mar/2024 14:03:34] "GET / HTTP/1.1" 200 -]
[2024-03-01 14:03:34,064: INFO: _internal: 127.0.0.1 - - [01/Mar/2024 14:03:34] "[33mGET /kidney-image.jpg HTTP/1.1[0m" 404 -]
[2024-03-01 14:03:34,371: INFO: _internal: 127.0.0.1 - - [01/Mar/2024 14:03:34] "[33mGET /kidney-image.jpg HTTP/1.1[0m" 404 -]
[2024-03-01 14:03:43,154: INFO: _internal: [31m[1mWARNING: This is a development server. Do not use it in a production deployment. Use a production WSGI server instead.[0m
 * Running on all addresses (0.0.0.0)
 * Running on http://127.0.0.1:8080
 * Running on http://10.2.12.16:8080]
[2024-03-01 14:03:43,154: INFO: _internal: [33mPress CTRL+C to quit[0m]
[2024-03-01 14:03:44,075: INFO: _internal: 127.0.0.1 - - [01/Mar/2024 14:03:44] "GET / HTTP/1.1" 200 -]
[2024-03-01 14:03:44,092: INFO: _internal: 127.0.0.1 - - [01/Mar/2024 14:03:44] "[33mGET /kidney-image.jpg HTTP/1.1[0m" 404 -]
[2024-03-01 14:03:44,392: INFO: _internal: 127.0.0.1 - - [01/Mar/2024 14:03:44] "[33mGET /kidney-image.jpg HTTP/1.1[0m" 404 -]
[2024-03-01 14:08:51,510: INFO: _internal: [31m[1mWARNING: This is a development server. Do not use it in a production deployment. Use a production WSGI server instead.[0m
 * Running on all addresses (0.0.0.0)
 * Running on http://127.0.0.1:8080
 * Running on http://10.2.12.16:8080]
[2024-03-01 14:08:51,510: INFO: _internal: [33mPress CTRL+C to quit[0m]
[2024-03-01 14:09:02,274: INFO: _internal: 127.0.0.1 - - [01/Mar/2024 14:09:02] "GET / HTTP/1.1" 200 -]
[2024-03-01 14:09:02,294: INFO: _internal: 127.0.0.1 - - [01/Mar/2024 14:09:02] "[33mGET /kidney-image.jpg HTTP/1.1[0m" 404 -]
[2024-03-01 14:09:02,613: INFO: _internal: 127.0.0.1 - - [01/Mar/2024 14:09:02] "[33mGET /kidney-image.jpg HTTP/1.1[0m" 404 -]
[2024-03-01 14:09:48,814: INFO: _internal: [31m[1mWARNING: This is a development server. Do not use it in a production deployment. Use a production WSGI server instead.[0m
 * Running on all addresses (0.0.0.0)
 * Running on http://127.0.0.1:8080
 * Running on http://10.2.12.16:8080]
[2024-03-01 14:09:48,815: INFO: _internal: [33mPress CTRL+C to quit[0m]
[2024-03-01 14:09:49,098: INFO: _internal: 127.0.0.1 - - [01/Mar/2024 14:09:49] "GET / HTTP/1.1" 200 -]
[2024-03-01 14:09:49,118: INFO: _internal: 127.0.0.1 - - [01/Mar/2024 14:09:49] "[33mGET /kidney-image.jpg HTTP/1.1[0m" 404 -]
[2024-03-01 14:09:49,383: INFO: _internal: 127.0.0.1 - - [01/Mar/2024 14:09:49] "[33mGET /kidney-image.jpg HTTP/1.1[0m" 404 -]
[2024-03-01 14:10:54,927: INFO: _internal: [31m[1mWARNING: This is a development server. Do not use it in a production deployment. Use a production WSGI server instead.[0m
 * Running on all addresses (0.0.0.0)
 * Running on http://127.0.0.1:8080
 * Running on http://10.2.12.16:8080]
[2024-03-01 14:10:54,927: INFO: _internal: [33mPress CTRL+C to quit[0m]
[2024-03-01 14:10:55,430: INFO: _internal: 127.0.0.1 - - [01/Mar/2024 14:10:55] "GET / HTTP/1.1" 200 -]
[2024-03-01 14:10:55,449: INFO: _internal: 127.0.0.1 - - [01/Mar/2024 14:10:55] "[33mGET /kidney-image.jpg HTTP/1.1[0m" 404 -]
[2024-03-01 14:10:55,607: INFO: _internal: 127.0.0.1 - - [01/Mar/2024 14:10:55] "[33mGET /kidney-image.jpg HTTP/1.1[0m" 404 -]
>>>>>>> 32d5db313161e85be616cf0ded7e9aa3c5b5036c
[2024-03-02 14:20:56,985: INFO: _internal: [31m[1mWARNING: This is a development server. Do not use it in a production deployment. Use a production WSGI server instead.[0m
 * Running on all addresses (0.0.0.0)
 * Running on http://127.0.0.1:8080
 * Running on http://172.16.0.213:8080]
[2024-03-02 14:20:56,985: INFO: _internal: [33mPress CTRL+C to quit[0m]
[2024-03-02 14:21:56,227: INFO: _internal: 127.0.0.1 - - [02/Mar/2024 14:21:56] "GET / HTTP/1.1" 200 -]
[2024-03-02 14:22:25,010: INFO: _internal: 127.0.0.1 - - [02/Mar/2024 14:22:25] "POST /predict HTTP/1.1" 200 -]
[2024-03-02 14:25:59,657: INFO: _internal: 127.0.0.1 - - [02/Mar/2024 14:25:59] "GET / HTTP/1.1" 200 -]
[2024-03-02 14:26:09,372: INFO: _internal: 127.0.0.1 - - [02/Mar/2024 14:26:09] "POST /predict HTTP/1.1" 200 -]
[2024-03-02 14:51:14,703: INFO: _internal: [31m[1mWARNING: This is a development server. Do not use it in a production deployment. Use a production WSGI server instead.[0m
 * Running on all addresses (0.0.0.0)
 * Running on http://127.0.0.1:8080
 * Running on http://172.16.0.213:8080]
[2024-03-02 14:51:14,703: INFO: _internal: [33mPress CTRL+C to quit[0m]
[2024-03-02 14:51:22,145: INFO: _internal: 127.0.0.1 - - [02/Mar/2024 14:51:22] "GET / HTTP/1.1" 200 -]
[2024-03-02 14:51:29,741: ERROR: app: Exception on /predict [POST]]
Traceback (most recent call last):
  File "/scratch/Shaz/miniconda3/envs/kidney2/lib/python3.12/site-packages/flask/app.py", line 1463, in wsgi_app
    response = self.full_dispatch_request()
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/scratch/Shaz/miniconda3/envs/kidney2/lib/python3.12/site-packages/flask/app.py", line 872, in full_dispatch_request
    rv = self.handle_user_exception(e)
         ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/scratch/Shaz/miniconda3/envs/kidney2/lib/python3.12/site-packages/flask_cors/extension.py", line 176, in wrapped_function
    return cors_after_request(app.make_response(f(*args, **kwargs)))
                                                ^^^^^^^^^^^^^^^^^^
  File "/scratch/Shaz/miniconda3/envs/kidney2/lib/python3.12/site-packages/flask/app.py", line 870, in full_dispatch_request
    rv = self.dispatch_request()
         ^^^^^^^^^^^^^^^^^^^^^^^
  File "/scratch/Shaz/miniconda3/envs/kidney2/lib/python3.12/site-packages/flask/app.py", line 855, in dispatch_request
    return self.ensure_sync(self.view_functions[rule.endpoint])(**view_args)  # type: ignore[no-any-return]
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/scratch/Shaz/miniconda3/envs/kidney2/lib/python3.12/site-packages/flask_cors/decorator.py", line 130, in wrapped_function
    resp = make_response(f(*args, **kwargs))
                         ^^^^^^^^^^^^^^^^^^
  File "/scratch/Shaz/subsidiary_work/Kidney_Disease_Classification/app.py", line 36, in predictRoute
    result = clApp.classifier.predict()
             ^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/scratch/Shaz/subsidiary_work/Kidney_Disease_Classification/src/cnn_classifier/pipeline/prediction.py", line 12, in predict
    model = load_model(os.path.join("model","model.h5"))
            ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/scratch/Shaz/miniconda3/envs/kidney2/lib/python3.12/site-packages/keras/src/saving/saving_api.py", line 183, in load_model
    return legacy_h5_format.load_model_from_hdf5(filepath)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/scratch/Shaz/miniconda3/envs/kidney2/lib/python3.12/site-packages/keras/src/legacy/saving/legacy_h5_format.py", line 155, in load_model_from_hdf5
    **saving_utils.compile_args_from_training_config(
      ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/scratch/Shaz/miniconda3/envs/kidney2/lib/python3.12/site-packages/keras/src/legacy/saving/saving_utils.py", line 143, in compile_args_from_training_config
    loss = _deserialize_nested_config(losses.deserialize, loss_config)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/scratch/Shaz/miniconda3/envs/kidney2/lib/python3.12/site-packages/keras/src/legacy/saving/saving_utils.py", line 202, in _deserialize_nested_config
    return deserialize_fn(config)
           ^^^^^^^^^^^^^^^^^^^^^^
  File "/scratch/Shaz/miniconda3/envs/kidney2/lib/python3.12/site-packages/keras/src/losses/__init__.py", line 124, in deserialize
    return serialization_lib.deserialize_keras_object(
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/scratch/Shaz/miniconda3/envs/kidney2/lib/python3.12/site-packages/keras/src/saving/serialization_lib.py", line 576, in deserialize_keras_object
    return deserialize_keras_object(
           ^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/scratch/Shaz/miniconda3/envs/kidney2/lib/python3.12/site-packages/keras/src/saving/serialization_lib.py", line 711, in deserialize_keras_object
    instance = cls.from_config(inner_config)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/scratch/Shaz/miniconda3/envs/kidney2/lib/python3.12/site-packages/keras/src/losses/losses.py", line 34, in from_config
    return cls(**config)
           ^^^^^^^^^^^^^
  File "/scratch/Shaz/miniconda3/envs/kidney2/lib/python3.12/site-packages/keras/src/losses/losses.py", line 737, in __init__
    super().__init__(
  File "/scratch/Shaz/miniconda3/envs/kidney2/lib/python3.12/site-packages/keras/src/losses/losses.py", line 16, in __init__
    super().__init__(reduction=reduction, name=name)
  File "/scratch/Shaz/miniconda3/envs/kidney2/lib/python3.12/site-packages/keras/src/losses/loss.py", line 29, in __init__
    self.reduction = standardize_reduction(reduction)
                     ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/scratch/Shaz/miniconda3/envs/kidney2/lib/python3.12/site-packages/keras/src/losses/loss.py", line 77, in standardize_reduction
    raise ValueError(
ValueError: Invalid value for argument `reduction`. Expected one of {None, 'none', 'sum', 'sum_over_batch_size'}. Received: reduction=auto
[2024-03-02 14:51:29,744: INFO: _internal: 127.0.0.1 - - [02/Mar/2024 14:51:29] "[35m[1mPOST /predict HTTP/1.1[0m" 500 -]
[2024-03-02 14:55:45,291: INFO: _internal: [31m[1mWARNING: This is a development server. Do not use it in a production deployment. Use a production WSGI server instead.[0m
 * Running on all addresses (0.0.0.0)
 * Running on http://127.0.0.1:8080
 * Running on http://172.16.0.213:8080]
[2024-03-02 14:55:45,291: INFO: _internal: [33mPress CTRL+C to quit[0m]
[2024-03-02 14:55:46,468: INFO: _internal: 127.0.0.1 - - [02/Mar/2024 14:55:46] "GET / HTTP/1.1" 200 -]
[2024-03-02 14:55:57,664: ERROR: app: Exception on /predict [POST]]
Traceback (most recent call last):
  File "/scratch/Shaz/miniconda3/envs/kidney2/lib/python3.12/site-packages/flask/app.py", line 1463, in wsgi_app
    response = self.full_dispatch_request()
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/scratch/Shaz/miniconda3/envs/kidney2/lib/python3.12/site-packages/flask/app.py", line 872, in full_dispatch_request
    rv = self.handle_user_exception(e)
         ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/scratch/Shaz/miniconda3/envs/kidney2/lib/python3.12/site-packages/flask_cors/extension.py", line 176, in wrapped_function
    return cors_after_request(app.make_response(f(*args, **kwargs)))
                                                ^^^^^^^^^^^^^^^^^^
  File "/scratch/Shaz/miniconda3/envs/kidney2/lib/python3.12/site-packages/flask/app.py", line 870, in full_dispatch_request
    rv = self.dispatch_request()
         ^^^^^^^^^^^^^^^^^^^^^^^
  File "/scratch/Shaz/miniconda3/envs/kidney2/lib/python3.12/site-packages/flask/app.py", line 855, in dispatch_request
    return self.ensure_sync(self.view_functions[rule.endpoint])(**view_args)  # type: ignore[no-any-return]
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/scratch/Shaz/miniconda3/envs/kidney2/lib/python3.12/site-packages/flask_cors/decorator.py", line 130, in wrapped_function
    resp = make_response(f(*args, **kwargs))
                         ^^^^^^^^^^^^^^^^^^
  File "/scratch/Shaz/subsidiary_work/Kidney_Disease_Classification/app.py", line 36, in predictRoute
    result = clApp.classifier.predict()
             ^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/scratch/Shaz/subsidiary_work/Kidney_Disease_Classification/src/cnn_classifier/pipeline/prediction.py", line 12, in predict
    model = load_model(os.path.join("model","model.h5"))
            ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/scratch/Shaz/miniconda3/envs/kidney2/lib/python3.12/site-packages/keras/src/saving/saving_api.py", line 183, in load_model
    return legacy_h5_format.load_model_from_hdf5(filepath)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/scratch/Shaz/miniconda3/envs/kidney2/lib/python3.12/site-packages/keras/src/legacy/saving/legacy_h5_format.py", line 155, in load_model_from_hdf5
    **saving_utils.compile_args_from_training_config(
      ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/scratch/Shaz/miniconda3/envs/kidney2/lib/python3.12/site-packages/keras/src/legacy/saving/saving_utils.py", line 143, in compile_args_from_training_config
    loss = _deserialize_nested_config(losses.deserialize, loss_config)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/scratch/Shaz/miniconda3/envs/kidney2/lib/python3.12/site-packages/keras/src/legacy/saving/saving_utils.py", line 202, in _deserialize_nested_config
    return deserialize_fn(config)
           ^^^^^^^^^^^^^^^^^^^^^^
  File "/scratch/Shaz/miniconda3/envs/kidney2/lib/python3.12/site-packages/keras/src/losses/__init__.py", line 124, in deserialize
    return serialization_lib.deserialize_keras_object(
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/scratch/Shaz/miniconda3/envs/kidney2/lib/python3.12/site-packages/keras/src/saving/serialization_lib.py", line 576, in deserialize_keras_object
    return deserialize_keras_object(
           ^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/scratch/Shaz/miniconda3/envs/kidney2/lib/python3.12/site-packages/keras/src/saving/serialization_lib.py", line 711, in deserialize_keras_object
    instance = cls.from_config(inner_config)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/scratch/Shaz/miniconda3/envs/kidney2/lib/python3.12/site-packages/keras/src/losses/losses.py", line 34, in from_config
    return cls(**config)
           ^^^^^^^^^^^^^
  File "/scratch/Shaz/miniconda3/envs/kidney2/lib/python3.12/site-packages/keras/src/losses/losses.py", line 737, in __init__
    super().__init__(
  File "/scratch/Shaz/miniconda3/envs/kidney2/lib/python3.12/site-packages/keras/src/losses/losses.py", line 16, in __init__
    super().__init__(reduction=reduction, name=name)
  File "/scratch/Shaz/miniconda3/envs/kidney2/lib/python3.12/site-packages/keras/src/losses/loss.py", line 29, in __init__
    self.reduction = standardize_reduction(reduction)
                     ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/scratch/Shaz/miniconda3/envs/kidney2/lib/python3.12/site-packages/keras/src/losses/loss.py", line 77, in standardize_reduction
    raise ValueError(
ValueError: Invalid value for argument `reduction`. Expected one of {None, 'sum_over_batch_size', 'sum', 'none'}. Received: reduction=auto
[2024-03-02 14:55:57,667: INFO: _internal: 127.0.0.1 - - [02/Mar/2024 14:55:57] "[35m[1mPOST /predict HTTP/1.1[0m" 500 -]
[2024-03-02 14:56:53,660: INFO: _internal: 127.0.0.1 - - [02/Mar/2024 14:56:53] "GET / HTTP/1.1" 200 -]
[2024-03-02 14:57:01,954: ERROR: app: Exception on /predict [POST]]
Traceback (most recent call last):
  File "/scratch/Shaz/miniconda3/envs/kidney2/lib/python3.12/site-packages/flask/app.py", line 1463, in wsgi_app
    self.add_template_global(f, name=name)
                   ^^^^^^^^^^^^^^^^^^^^^^^^
  File "/scratch/Shaz/miniconda3/envs/kidney2/lib/python3.12/site-packages/flask/app.py", line 872, in full_dispatch_request
    def templates_auto_reload(self) -> bool:
                 ^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/scratch/Shaz/miniconda3/envs/kidney2/lib/python3.12/site-packages/flask_cors/extension.py", line 176, in wrapped_function
    return cors_after_request(app.make_response(f(*args, **kwargs)))
                                                ^^^^^^^^^^^^^^^^^^
  File "/scratch/Shaz/miniconda3/envs/kidney2/lib/python3.12/site-packages/flask/app.py", line 870, in full_dispatch_request
  File "/scratch/Shaz/miniconda3/envs/kidney2/lib/python3.12/site-packages/flask/app.py", line 855, in dispatch_request
    if prefix is None:
           ^^^^^^^^^^^^
  File "/scratch/Shaz/miniconda3/envs/kidney2/lib/python3.12/site-packages/flask_cors/decorator.py", line 130, in wrapped_function
    resp = make_response(f(*args, **kwargs))
                         ^^^^^^^^^^^^^^^^^^
  File "/scratch/Shaz/subsidiary_work/Kidney_Disease_Classification/app.py", line 36, in predictRoute
    result = clApp.classifier.predict()
             ^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/scratch/Shaz/subsidiary_work/Kidney_Disease_Classification/src/cnn_classifier/pipeline/prediction.py", line 12, in predict
    model = load_model(os.path.join("model","model.h5"))
            ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/scratch/Shaz/miniconda3/envs/kidney2/lib/python3.12/site-packages/keras/src/saving/saving_api.py", line 183, in load_model
    return legacy_h5_format.load_model_from_hdf5(filepath)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/scratch/Shaz/miniconda3/envs/kidney2/lib/python3.12/site-packages/keras/src/legacy/saving/legacy_h5_format.py", line 155, in load_model_from_hdf5
    **saving_utils.compile_args_from_training_config(
      ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/scratch/Shaz/miniconda3/envs/kidney2/lib/python3.12/site-packages/keras/src/legacy/saving/saving_utils.py", line 143, in compile_args_from_training_config
    loss = _deserialize_nested_config(losses.deserialize, loss_config)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/scratch/Shaz/miniconda3/envs/kidney2/lib/python3.12/site-packages/keras/src/legacy/saving/saving_utils.py", line 202, in _deserialize_nested_config
    return deserialize_fn(config)
           ^^^^^^^^^^^^^^^^^^^^^^
  File "/scratch/Shaz/miniconda3/envs/kidney2/lib/python3.12/site-packages/keras/src/losses/__init__.py", line 124, in deserialize
    return serialization_lib.deserialize_keras_object(
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/scratch/Shaz/miniconda3/envs/kidney2/lib/python3.12/site-packages/keras/src/saving/serialization_lib.py", line 576, in deserialize_keras_object
    return deserialize_keras_object(
           ^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/scratch/Shaz/miniconda3/envs/kidney2/lib/python3.12/site-packages/keras/src/saving/serialization_lib.py", line 711, in deserialize_keras_object
    instance = cls.from_config(inner_config)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/scratch/Shaz/miniconda3/envs/kidney2/lib/python3.12/site-packages/keras/src/losses/losses.py", line 34, in from_config
    return cls(**config)
           ^^^^^^^^^^^^^
  File "/scratch/Shaz/miniconda3/envs/kidney2/lib/python3.12/site-packages/keras/src/losses/losses.py", line 737, in __init__
    super().__init__(
  File "/scratch/Shaz/miniconda3/envs/kidney2/lib/python3.12/site-packages/keras/src/losses/losses.py", line 16, in __init__
    super().__init__(reduction=reduction, name=name)
  File "/scratch/Shaz/miniconda3/envs/kidney2/lib/python3.12/site-packages/keras/src/losses/loss.py", line 29, in __init__
    self.reduction = standardize_reduction(reduction)
                     ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/scratch/Shaz/miniconda3/envs/kidney2/lib/python3.12/site-packages/keras/src/losses/loss.py", line 77, in standardize_reduction
    raise ValueError(
ValueError: Invalid value for argument `reduction`. Expected one of {None, 'sum_over_batch_size', 'sum', 'none'}. Received: reduction=auto
[2024-03-02 14:57:01,956: INFO: _internal: 127.0.0.1 - - [02/Mar/2024 14:57:01] "[35m[1mPOST /predict HTTP/1.1[0m" 500 -]
[2024-03-02 14:59:00,943: INFO: _internal: [31m[1mWARNING: This is a development server. Do not use it in a production deployment. Use a production WSGI server instead.[0m
 * Running on all addresses (0.0.0.0)
 * Running on http://127.0.0.1:8080
 * Running on http://172.16.0.213:8080]
[2024-03-02 14:59:00,943: INFO: _internal: [33mPress CTRL+C to quit[0m]
[2024-03-02 14:59:03,325: INFO: _internal: 127.0.0.1 - - [02/Mar/2024 14:59:03] "GET / HTTP/1.1" 200 -]
[2024-03-02 14:59:11,312: INFO: _internal: 127.0.0.1 - - [02/Mar/2024 14:59:11] "POST /predict HTTP/1.1" 200 -]
[2024-03-02 15:04:42,793: INFO: _internal: 127.0.0.1 - - [02/Mar/2024 15:04:42] "GET / HTTP/1.1" 200 -]
[2024-03-02 15:05:01,106: INFO: _internal: 127.0.0.1 - - [02/Mar/2024 15:05:01] "POST /predict HTTP/1.1" 200 -]
[2024-03-02 15:13:17,121: INFO: _internal: 127.0.0.1 - - [02/Mar/2024 15:13:17] "GET / HTTP/1.1" 200 -]
[2024-03-02 15:13:22,518: INFO: _internal: 127.0.0.1 - - [02/Mar/2024 15:13:22] "POST /predict HTTP/1.1" 200 -]
[2024-03-02 15:13:38,894: INFO: _internal: [31m[1mWARNING: This is a development server. Do not use it in a production deployment. Use a production WSGI server instead.[0m
 * Running on all addresses (0.0.0.0)
 * Running on http://127.0.0.1:8080
 * Running on http://172.16.0.213:8080]
[2024-03-02 15:13:38,894: INFO: _internal: [33mPress CTRL+C to quit[0m]
[2024-03-02 15:13:42,176: INFO: _internal: 127.0.0.1 - - [02/Mar/2024 15:13:42] "GET / HTTP/1.1" 200 -]
[2024-03-02 15:13:52,657: INFO: _internal: 127.0.0.1 - - [02/Mar/2024 15:13:52] "POST /predict HTTP/1.1" 200 -]
[2024-03-02 15:15:22,979: INFO: _internal: 127.0.0.1 - - [02/Mar/2024 15:15:22] "GET / HTTP/1.1" 200 -]
[2024-03-02 15:15:47,263: INFO: _internal: 127.0.0.1 - - [02/Mar/2024 15:15:47] "POST /predict HTTP/1.1" 200 -]
